ci.py

```diff
@@ -45,6 +48,37 @@
     'monasca-notification': 'monasca-notification',
     'grafana-init': 'grafana-init'
 }
+LOGS_PIPELINE_MODULE_TO_COMPOSE_SERVICES = {
+    'monasca-log-metrics': 'log-metrics',
+    'monasca-log-persister': 'log-persister',
+    'monasca-log-transformer': 'log-transformer',
+    'elasticsearch-init': 'elasticsearch-init',
+    'kafka-init': 'kafka-log-init',
+    'kibana': 'kibana',
+    'monasca-log-api': 'log-api',
+}
```

Comment:
I think this is the correct way to do this at the moment, but I'd like to move all this hardcoded metadata elsewhere at some point... maybe as metadata fields in build.yml or something. Feels a bit ugly especially as we add new components.